_sass/vars.scss

```diff
@@ -33,6 +33,8 @@ $font_weight-bold: 700;
 $color_navy: #101436;
 $color-bg_gray: #CDCDCD;
 $color_gold: #DB9E00;
+$color_light_gold: #FDF9F0;
```

Comment:
I think that this is actually supposed to be #FCFCAB (a la Jessie's comment on Ryan's list of remaining features in Slack)

Reply:
I think she was referring to the map coloring not the background gold.